src/index.html

```diff
@@ -15,8 +15,99 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="stylesheet"
+      href="https://fonts.google.com/specimen/Roboto"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="Apple"
+        class="img"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo"
+        />
+      </a>
+
+      <nav>
+        <ul class="nav__list">
+          <li>
+            <a
+              href="Apple"
+              class="nav__link link__marked is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="Samsung"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="Smartphones"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="Laptops & Computers"
```

Comment:
The href attribute 'Laptops & Computers' is not a valid URL. It should be a valid URL or path, such as '/laptops-computers' or 'https://example.com/laptops-computers'.